packages/Webkul/Installer/src/Console/Commands/Installer.php

```diff
@@ -182,43 +182,49 @@ protected function askForApplicationDetails()
 
         $this->info('Your Default Timezone is ' . date_default_timezone_get());
 
+        do {
+            $allowedLocales = $this->allowedChoice(
+                'Please choose the <bg=black;fg=white;options=bold>allowed locales</> for your channels',
+                $this->locales,
+                ['en' => $this->locales['en']],
+                10
+            );
+        
+            if (empty($allowedLocales)) {
+                $this->error('At least one locale is required. Please choose again.');
+            }
+        } while (empty($allowedLocales));
+
+        do {
+            $allowedCurrencies = $this->allowedChoice(
+                'Please choose the <bg=black;fg=white;options=bold>allowed currencies</> for your channels',
+                $this->currencies,
+                ['USD' => $this->currencies['USD']],
+                10
+            );
+        
+            if (empty($allowedCurrencies)) {
+                $this->error('At least one currency is required. Please choose again.');
+            }
+        } while (empty($allowedCurrencies));
+
         $defaultLocale = $this->updateEnvChoice(
             'APP_LOCALE',
-            'Please select the <bg=green>default application locale</>',
-            $this->locales
+            'Please select the <bg=black;fg=white;options=bold>default application locale</>',
+            $allowedLocales
         );
 
         $defaultCurrency = $this->updateEnvChoice(
             'APP_CURRENCY',
-            'Please select the <bg=green>default currency</>',
-            $this->currencies
+            'Please select the <bg=black;fg=white;options=bold>default currency</>',
+            $allowedCurrencies
         );
```

Comment:
Same here,

This is the admin panel currency which is set in the .env file, not in the channel default currency.

Reply:
Reverted in the latest commit.